src/App.jsx

```diff
@@ -1,11 +1,28 @@
 import './App.css'
-import logo from './assets/logo-white.png'
+// import logo from './assets/logo-white.png'
+import {Routes, Route} from 'react-router-dom';
+import Home from "./pages/home/Home.jsx";
+import NieuwePost from "./pages/nieuwePost/NieuwePost.jsx";
+import Overzicht from "./pages/overzicht/Overzicht.jsx";
+import Page404 from "./pages/page404/Page404.jsx";
+import React from "react";
+import Navigation from "./components/navigation/Navigation.jsx";
+import Posts from "./components/posts/Posts.jsx";
+
 
 function App() {
     return (
         <div className="page-container">
-            <img src={logo} alt="Company logo"/>
-            <h1>Begin hier met het maken van jouw blog-applicatie!</h1>
+
+            <Navigation/>
+            <Routes>
+                <Route path='/' element={<Home />}/>
+                <Route path='/nieuwePost' element={<NieuwePost />}/>
```

Comment:
Je heb voor de pagina's Nederlandse namen gebruik, maar het is conventie om code in engels te schrijven.